templates/openstack/compare.html

```diff
@@ -1,432 +1,548 @@
-      <p>
-        <a href="https://assets.ubuntu.com/v1/a08e868a-Charmed+OpenStack_15.03.21.pdf">
-          Learn more in our datasheet&nbsp;&rsaquo;
-        </a>
-      </p>
-    </div>
-  </div>
-</section>
+            <tr>
+              <th>Containerised control plane</th>
+              <td class="p-accordion is-compact ">
+                <button class="p-accordion__tab "
+                        id="tab-control-plane"
+                        role="tab"
+                        aria-controls="content-control-plane"
+                        aria-expanded="false">
+                  <img src="https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg"
+                       alt="Yes" />
+                </button>
+                <div class="p-accordion__panel"
+                     aria-hidden="true"
+                     id="content-control-plane">
+                  <p>
+                    Containers bring a lot of advantages over traditional virtual machines such as better density and management capabilities. The control plane of Charmed OpenStack is fully containerised using <a href="/lxd">LXD</a> containers.
+                  </p>
+                </div>
+              </td>
+              <td>
+                <img src="https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg"
+                     alt="Yes" />
+              </td>
+              <td>-</td>
+            </tr>
 
-<section class="p-strip">
-  <div class="row u-equal-height">
-    <div class="col-8">
-      <h2>Open source unlike VMware</h2>
-      <p>Canonical's Charmed OpenStack is fully open source which allows it to benefit from a broader community, faster development process and wider selection of technology choices than those available in VMware's platform. Choosing an open source platform avoids vendor lock-in and significantly reduces infrastructure costs.</p>
-      <p><a href="/engage/vmware-to-charmed-openstack">Explore VMware to OpenStack migration options&nbsp;&rsaquo;</a></p>
-    </div>
-    <div class="col-4 u-vertically-center u-hide--medium u-hide--small">
-      {{
-        image(
-          url="https://assets.ubuntu.com/v1/eb3a0aa2-openstack-opensource.svg",
-          alt="",
-          width="386",
-          height="220",
-          hi_def=True,
-          loading="lazy",
-        ) | safe
-      }}
-    </div>
-  </div>
-</section>
+            <tr>
+              <th>Fully automated upgrades</th>
+              <td class="p-accordion is-compact ">
+                <button class="p-accordion__tab "
+                        id="tab-automated-upgrades"
+                        role="tab"
+                        aria-controls="content-automated-upgrades"
+                        aria-expanded="false">
+                  <img src="https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg"
+                       alt="Yes" />
```

Comment:
Why not use image template here?

Reply:
Adding SVGs to the image template seems to make them pixelated.
But the recent change to the image template [here](https://github.com/canonical/canonicalwebteam.image-template/pull/69) should fix that.
So I have added it to the image template.